packages/grafana-prometheus/src/querybuilder/parsing.test.ts

```diff
@@ -290,6 +290,27 @@ describe('buildVisualQueryFromString', () => {
     });
   });
 
+  it('parses a native histogram function correctly', () => {
+    expect(
+      buildVisualQueryFromString('histogram_count(rate(counters_logins{app="backend"}[$__rate_interval]))')
+    ).toEqual({
+      errors: [],
+      query: {
+        metric: 'counters_logins',
+        labels: [{ label: 'app', op: '=', value: 'backend' }],
+        operations: [
+          {
+            id: 'rate',
+            params: ['$__rate_interval'],
+          },
+          {
+            id: 'histogram_quantile',
+          },
+        ],
+      },
+    });
+  });
+
```

Comment:
This will always fail because `lezer-promqlv0.37.0-rc.1` has no idea about new histogram functions. So you can skip it and I'll enable it in [my PR](https://github.com/grafana/grafana/pull/85942)

Reply:
I just tried your changes on my PR and it works!


https://github.com/grafana/grafana/assets/820946/9f10c25d-2708-40b6-b6d1-50efc020c815

Reply:
Of course there is no graph available because the metric I used is not native histogram. I picked it just for the sake of the demo.

Reply:
Yes! I wrote it just to have a test ready for when we do get it in. I will remove it and you can add it to your PR. Go Lezer parser update!!! Great job Ismail!!! 🏆

Reply:
I would say don't remove it. But just mark it to skip. You can define it as 
`xit('parses a native...` so it will be ignored
^^^___________________________________